tools/its.py

```diff
@@ -255,9 +255,15 @@ def driver_enter_code(self, driver, plz_impfzentrum):
         # Maus-Bewegung hinzufügen (nicht sichtbar)
         action.move_by_offset(10, 20).perform()
         time.sleep(randint(1, 3))
-        action.move_by_offset(randint(1, 9), randint(1, 9)).perform()
+        try:
+            action.move_by_offset(randint(1, 100), randint(1, 100)).perform()
```

Comment:
Man muss halt aufpassen, dass nicht aus versehen irgendwo auf einen Link geklickt wird

Reply:
Wäre ja nicht schlimm oder?

Reply:
ah ne stimmt, da hatte ich einen Denkfehler, die Cookies bleiben ja erhalten, selbst wenn man auf die Startseite weitergeleitet wird ^^

Jedoch hätte ich evtl. noch einen anderen Verbesserungsvorschlag
Du könntest das ganze ja in einer Schleife machen, dann bräuchtest du nicht so viele Zeilen und nur 1 try / except
```
for i in range(3):
   try:
        action.move_by_offset(randint(1, 100), randint(1, 100)).perform()
        time.sleep(randint(1, 3))
   except Exception as error:
        #evt fehler in den Logs speichern, damit man weiß was los war
        pass
```